src/features/CourtCaseFilters/AppliedFilters.tsx

```diff
@@ -0,0 +1,64 @@
+import If from "components/If"
+import { useRouter } from "next/router"
+import { Filter } from "types/CaseListQueryParams"
+import deleteQueryParam from "utils/deleteQueryParam"
+
+interface Props {
+  courtCaseTypes: Filter[]
+  keywords: string[]
+  urgency?: boolean
+}
+
+const AppliedFilters: React.FC<Props> = ({ courtCaseTypes, keywords, urgency }: Props) => {
+  const { basePath, query } = useRouter()
+
+  const hasAnyAppliedFilters = () => courtCaseTypes.length > 0 || keywords.length > 0 || urgency
+
+  const removeQueryParamFromPath = (paramToRemove: { [key: string]: string }): string => {
+    const searchParams = deleteQueryParam(paramToRemove, query)
+
+    return `${basePath}/?${searchParams}`
+  }
+
+  return (
+    <div>
+      <If condition={hasAnyAppliedFilters()}>
+        <ul key={"applied-filters"} className="moj-filter-tags">
+          <li>
+            <h3 className="govuk-heading-s govuk-!-margin-bottom-0">{"Applied filters:"}</h3>
+          </li>
+          {courtCaseTypes.map((t) => {
+            return (
+              <li key={`${t}`}>
+                <a className="moj-filter__tag" href={removeQueryParamFromPath({ type: t })}>
+                  <span className="govuk-visually-hidden">{`Remove ${t} filter`}</span>
+                  {t}
+                </a>
+              </li>
```

Comment:
It seems like the tag markup is a good candidate for being a separate component.
It also reduces the code in this file as well.

Reply:
Great advise, resolved in [31f0afe](https://github.com/ministryofjustice/bichard7-next-ui/pull/98/commits/31f0afea9791873c00fa09917ef50cbb00381001)